### ISSUE TITLE
Questions about the pretrain model and officehome lable.

### ISSUE THREAD

**Issue by zhouzhou9999** [thuml/Calibrated-Multiple-Uncertainties#3]
hello！Can you share the pre-trained resnet-50 model and officehome label files?
Very urgent and thanks!

**Comment by zhouzhou9999** [comment]
I have downloaded the resnet-50 model on the Internet（https://download.pytorch.org/models/resnet50-19c8e357.pth）, but it is not very feasible
